templates/monitor/stats.html

```diff
@@ -0,0 +1,57 @@
+{% extends "base.html" %}
+
+{% block extra-head %}
+<style type="text/css">
+  .tags{font-family: Arial, sans-serif;font-size:10px;}
```

Comment:
please choose readability over this format.

also, make sure your formatting is consistent.